### ISSUE TITLE
Add automation for projects

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse#1168]
## Description

Via #1167 we've added some project-specific labels. We can use these labels to automatically add certain items to the various boards, namely:

- Add the appropriate label ([project proposal](https://github.com/WordPress/openverse/labels/%F0%9F%A7%AD%20project%3A%20proposal), [implementation plan](https://github.com/WordPress/openverse/labels/%F0%9F%A7%AD%20project%3A%20implementation%20plan)) to PRs which affect the `rfc/` folder (or `documentation/projects/proposals` after #1164) based on filename
- Add PRs labeled with the above to the [discussion board](https://github.com/orgs/WordPress/projects/79)

## Additional context

This was discussed during the [weekly dev meeting](https://wordpress.slack.com/archives/C02012JB00N/p1681228425809849?thread_ts=1681227888.069269&cid=C02012JB00N).

**Comment by AetherUnbound** [comment]
I think 1 project might make sense, but we may also have discussions happening that _aren't_ tied to a specific project, so perhaps it makes sense to keep them separate for that reason?

**Comment by sarayourfriend** [comment]
Duplicate of #1931
